react/locales/en-US.json

```diff
@@ -1,14 +1,21 @@
 {
-  "search.text": "You searched for {search}",
-  "sortBy.text": "{quantity} product(s) sorted by",
-  "sortBy.relevance": "Relevance",
-  "sortBy.higherPrice": "Price: High to Low",
-  "sortBy.lowerPrice": "Price: Low to High",
-  "sortBy.nameAZ": "Name(A-Z)",
-  "sortBy.nameZA": "Name(Z-A)",
-  "sortBy.bestSellers": "Best sellers",
-  "editor.gallery.title": "Gallery",
-  "editor.gallery.description": "A product gallery",
-  "editor.gallery.columnsQuantityLarge.title": "Columns quantity (large viewport)",
-  "editor.gallery.columnsQuantityMedium.title": "Columns quantity (medium viewport)"
-}
+  "Departments": "Departments",
```

Comment:
lowercase

Reply:
This key `Departments` comes from server. I do the intl bind directly by the key retrieved from the request.
The same occurs to Categories, Brands, Operational System, Memory (RAM)